update README: reorganize and dev instructions (#41)
terraref/brapi: README.md

> | /germplasm  | cultivars.  |       | 

Reviewer: Is the period at the end of cultivars a typo?

| /observations | traits | |

## How to set up a development environment.

We assume you have the following installed:

> - Docker (to run BETYdb)

Reviewer: ```suggestion
- [Docker](https://www.docker.com/products/docker-desktop) (to run BETYdb)
```

- [Postman](https://www.postman.com/) (to test out APIs)
- [PyCharm](https://www.jetbrains.com/pycharm/) (for development)

### Run a development instance of BETYdb

> The following commands can be used to initialize the database to be used with BRAPI development. This will fetch public data from TERRA-REF database (id=6). To get the latest data you can call the sync command. 

Reviewer: ```suggestion
The following commands can be used to initialize the database to be used with BRAPI development. Run these commands from within the root directory of a this repository. These commands will fetch public data from the TERRA-REF instance of the BETYdb database (id=6). To get the latest data you can call the sync command. 
```

```bash
docker-compose up -d postgres
docker-compose run --rm bety initialize
docker-compose run --rm bety sync
```

_Note:_ If you get an error about port 5432 already in use, you likely need to stop another instance of postgres (locally or as docker).

### Set up PyCharm 

Now that you have BETYdb running on port 5432, you will be able to begin developing the server. These instructions assume that you are using PyCharm and have a project open to your brapi directory.

* Go to run --> edit configurations
* Set the "name" field to "server"

> * Set the 'script path' to `<localpath>/brapi/server.py`

Reviewer: ```suggestion
* Set the 'script path' to `<localpath>/brapi/server.py` (localpath is file path to local `server.py`) 
```

> * In the top right of PyCharm, select the server configuration and click the green triangle to run the server

Reviewer: I think it selects the new configuration automatically, so I might replace "select the server configuration" to "make sure server configuration `server` is selected".

Author: updated -

> * In the top right of PyCharm, select the server configuration and click the green triangle to run the server

Author: ```suggestion
* In the top right of PyCharm, make sure the configuration server is selected. Then click the green triangle to run the server
```

* In Postman test that the API exists by navigating to http://localhost:5000/brapi/v1/calls  

> 

Reviewer: ```suggestion

When you open `requirements.txt` in PyCharm, you will be prompted to install any required Python packages that are not already installed. 

```
